plugins.py

```diff
@@ -1,34 +1,64 @@
 #!/bin/env python
 
 import os
-from shutil import copy
+import cgi
+import urllib.request
+import random
+import string
+from shutil import copyfileobj
 from distutils.dir_util import copy_tree
 from distutils import log
 
-pluginsKey = "SSO_PLUGINS"
-pluginsVolume = "/opt/plugins"
+extensionsKey = "KEYCLOAK_EXTENSIONS"
+extensionsVolume = "/opt/extensions"
```

Comment:
Was it mentioned that the path for RH-SSO was different to Keycloak?

Reply:
Yes, the path for RHSSO is different but that is only relevant on the operator side. The correct volume gets always mounted to the init container under this path.

Reply:
ok, thanks @pb82